airgun/views/oscapreports.py

```diff
@@ -0,0 +1,47 @@
+from widgetastic.widget import Checkbox
+from widgetastic.widget import Table
+from widgetastic.widget import Text
+
+from airgun.views.common import BaseLoggedInView
+from airgun.views.common import SearchableViewMixin
+from airgun.widgets import ActionsDropdown
+
+
+class OSCAPReportsView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h1[text()='Compliance Reports']")
+    delete_reports = Text("//div[@id='submit_multiple']")
```

Comment:
Technically, this is button group and not button itself. But right now this group contains only one member.
That theoretically matters, as button group dimensions on page are slightly bigger than button dimensions. So, theoretically, driver could click on area that is not button and not trigger action.

Also, buttons can be modeled by `Button` from `widgetastic_patternfly`.

Practically, neither of these really matters.